astropy/config/tests/test_configs.py

```diff
@@ -470,7 +470,9 @@ def get_content(fn):
 
 class TestAliasRead:
     def setup_class(self):
-        configuration._override_config_file = get_pkg_data_filename("data/alias.cfg")
+        configuration._override_config_file = str(
```

Comment:
just to confirm, the plan is to have a followup PR that means we don't need to do str casting?

Reply:
Yes!